<ui version="4.0" >
 <comment>*********************************************************************
**
** Copyright (C) 1992-2008 Trolltech ASA. All rights reserved.
**
** This file is part of the Qt Designer of the Qt Toolkit.
**
** This file may be used under the terms of the GNU General Public
** License versions 2.0 or 3.0 as published by the Free Software
** Foundation and appearing in the files LICENSE.GPL2 and LICENSE.GPL3
** included in the packaging of this file.  Alternatively you may (at
** your option) use any later version of the GNU General Public
** License if such license has been publicly approved by Trolltech ASA
** (or its successors, if any) and the KDE Free Qt Foundation. In
** addition, as a special exception, Trolltech gives you certain
** additional rights. These rights are described in the Trolltech GPL
** Exception version 1.2, which can be found at
** http://www.trolltech.com/products/qt/gplexception/ and in the file
** GPL_EXCEPTION.txt in this package.
**
** Please review the following information to ensure GNU General
** Public Licensing requirements will be met:
** http://trolltech.com/products/qt/licenses/licensing/opensource/. If
** you are unsure which license is appropriate for your use, please
** review the following information:
** http://trolltech.com/products/qt/licenses/licensing/licensingoverview
** or contact the sales department at sales@trolltech.com.
**
** In addition, as a special exception, Trolltech, as the sole
** copyright holder for Qt Designer, grants users of the Qt/Eclipse
** Integration plug-in the right for the Qt/Eclipse Integration to
** link to functionality provided by Qt Designer and its related
** libraries.
**
** This file is provided "AS IS" with NO WARRANTY OF ANY KIND,
** INCLUDING THE WARRANTIES OF DESIGN, MERCHANTABILITY AND FITNESS FOR
** A PARTICULAR PURPOSE. Trolltech reserves all rights not expressly
** granted herein.
**
** This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
** WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
**
*********************************************************************</comment>
 <class>qdesigner_internal::QtBrushEditor</class>
 <widget class="QWidget" name="qdesigner_internal::QtBrushEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>172</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="topLayoutWidget" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QToolButton" name="editToolButton" >
         <property name="text" >
          <string>Edit...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" >
        <widget class="QToolButton" name="gradientToolButton" >
         <property name="toolTip" >
          <string>Edit Gradient</string>
         </property>
         <property name="text" >
          <string>Gradient...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>26</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="5" >
        <widget class="QFrame" name="frame" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="qdesigner_internal::QtBrushWidget" name="brushWidget" />
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QToolButton" name="patternToolButton" >
         <property name="toolTip" >
          <string>Edit Color Pattern</string>
         </property>
         <property name="text" >
          <string>Color Pattern...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QToolButton" name="textureToolButton" >
         <property name="toolTip" >
          <string>Edit Texture</string>
         </property>
         <property name="text" >
          <string>Texture...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="bottomLayoutWidget" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" colspan="4" >
        <widget class="QListWidget" name="listWidget" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>160</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QToolButton" name="removeButton" >
         <property name="toolTip" >
          <string>Remove From Custom</string>
         </property>
         <property name="text" >
          <string>Remove From Custom</string>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>92</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2" >
        <widget class="QToolButton" name="addToCustomButton" >
         <property name="toolTip" >
          <string>Add To Custom</string>
         </property>
         <property name="text" >
          <string>Add To Custom</string>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QToolButton" name="applyButton" >
         <property name="toolTip" >
          <string>Apply Custom</string>
         </property>
         <property name="text" >
          <string>Apply</string>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qdesigner_internal::QtBrushWidget</class>
   <extends>QFrame</extends>
   <header>qtbrushwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>patternToolButton</tabstop>
  <tabstop>textureToolButton</tabstop>
  <tabstop>gradientToolButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
